### PR TITLE
NH-35608: Journal logs are not collected

### PR DESCRIPTION
**Pull request by gantrior**

### CODE REVIEW

File: tests/integration/test_utils.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ def get_all_log_resources(log_bulk):
     return result
 
 def get_all_bodies(log_bulk):
-    result = [records["body"]["stringValue"]
+    result = [records["body"]["stringValue"] if "stringValue" in records["body"] else records["body"]
```
Comment (pstranak-sw) [inline tests/integration/test_utils.py:15]: Why this change?

Comment (gantrior) [reply tests/integration/test_utils.py:15]: Because journal logs started to work in GitHub actions pipeline so started to appear in logs.json in mock service and integration test script was failing with the message that there is no `stringValue` on `body`. I assumed that journal logs has slightly different format than container logs. This condition fix it.

**Review by pstranak-sw — APPROVED** [review]
